docs/articles/faq/README.md

```diff
@@ -32,37 +32,33 @@ This proxy injects the driver script that emulates user actions into the tested
 Feel free to ask for more details.
 
-### What is the difference between a [paid](https://testcafe.devexpress.com) and an [open-source](https://devexpress.github.io/testcafe) TestCafe version? What is [TestCafe Studio](https://testcafe-studio.devexpress.com/)?
+### What is the difference between [TestCafe Studio](https://www.devexpress.com/products/testcafestudio/) and [open-source TestCafe](https://devexpress.github.io/testcafe)? Has [TestCafe v2015.1](https://testcafe.devexpress.com) been deprecated?
 
-All three versions share the same core features:
+Both TestCafe Studio and open-source TestCafe share the same core features:
 
 * No need for WebDriver, browser plugins or other tools.
 * Cross-platform and cross-browser out of the box.
 
-[TestCafe](https://testcafe.devexpress.com)  
-*first released in 2013, commercial web application*  
-
-* Visual Test Recorder and web GUI to create, edit and run tests.
-* You can record tests or edit them as JavaScript code.
-
 [TestCafe](https://devexpress.github.io/testcafe)  
-*first released in 2016,  free and open-source node.js application*
+*first released in 2016, free and open-source node.js application*
 
 * You can write tests in the latest JavaScript or TypeScript.
-* Clearer and more flexible [API](https://devexpress.github.io/testcafe/documentation/test-api/) supports ES6 and [PageModel pattern](https://devexpress.github.io/testcafe/documentation/recipes/using-page-model.html).
-* More stable tests due to the [Smart Assertion Query Mechanism](https://devexpress.github.io/testcafe/documentation/test-api/assertions/#smart-assertion-query-mechanism).
-* Tests run faster due to improved [Automatic Waiting Mechanism](https://devexpress.github.io/testcafe/documentation/test-api/waiting-for-page-elements-to-appear.html) and [Concurrent Test Execution](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/concurrent-test-execution.html).
+* Clear and flexible [API](https://devexpress.github.io/testcafe/documentation/test-api/) supports ES6 and [PageModel pattern](https://devexpress.github.io/testcafe/documentation/recipes/using-page-model.html).
+* Stable tests due to the [Smart Assertion Query Mechanism](https://devexpress.github.io/testcafe/documentation/test-api/assertions/#smart-assertion-query-mechanism).
+* Tests run fast due to intelligent [Automatic Waiting Mechanism](https://devexpress.github.io/testcafe/documentation/test-api/waiting-for-page-elements-to-appear.html) and [Concurrent Test Execution](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/concurrent-test-execution.html).
 * Easy integration: it is a node.js solution with CLI and reporters for popular CI systems.
 * You can extend it with [plugins](https://github.com/DevExpress/testcafe#plugins) and other Node.js modules.
 
-[TestCafe Studio](https://testcafe-studio.devexpress.com)  
+[TestCafe Studio](https://www.devexpress.com/products/testcafestudio/)  
```

Comment:
I would move TestCafe Studio as a first item in the answer before the open source TestCafe version (exchange their positions). 

A regular visitor of the TestCafe open source site likely knows what TestCafe open source version is. However, he unlikely knows what TestCafe Studio is (as it is a web site not related to Studio). Also, not everyone likes to read big texts, so it is better to mention TestCafe Studio first rather than a product for which the entire site is dedicated.